app/views/widgets/home_page/_hero_panel.html

```diff
@@ -9,13 +9,13 @@
   <section id="hero" class="dark-bg align-left" style="background-image:url('http://www.chapman.edu/_files/img/hero-panels/fallback-images/hilbert-art.jpg');">
     <div id="mastheadContent" class="maxWidth">
       <h2>
-        <span class="heading uppercase extended">California Scene Painting:</span>
-        <span class="subheading lowercase thin">A Narrative Revolution</span>
+        <span class="heading  fast_transition" style="position: relative; left: 0px; opacity: 1;">California Scene Painting:</span>
+        <span class="subheading thin  fast_transition" style="position: relative; left: 0px; opacity: 1;">A Narrative Revolution</span>
       </h2>
 
-      <p class="excerpt">The once overlooked California Scene painting movement gets the recognition it deserves and a new home at the Hilbert Museum of California Art at Chapman University.</p>
+      <p class="excerpt fast_transition" style="position: relative; left: 0px; opacity: 1;">The once overlooked California Scene painting movement gets the recognition it deserves and a new home at the Hilbert Museum of California Art at Chapman University.</p>
 
-      <div class="actions">
+      <div class="actions fast_transition style="position: relative; left: 0px; opacity: 1;"">
```

Comment:
It looks like a copy-paste error. If this is what is getting output to production, we should open a defect with a link to the page. In any event, the markup should be fixed here to avoid confusion in the future.